cockatrice/src/abstractcarddragitem.cpp

```diff
@@ -1,33 +1,42 @@
 #include "abstractcarddragitem.h"
 #include "carddatabase.h"
 #include <QCursor>
-#include <QGraphicsSceneMouseEvent>
 #include <QDebug>
+#include <QGraphicsSceneMouseEvent>
 #include <QPainter>
 
 static const float CARD_WIDTH_HALF = CARD_WIDTH / 2;
 static const float CARD_HEIGHT_HALF = CARD_HEIGHT / 2;
 const QColor GHOST_MASK = QColor(255, 255, 255, 50);
 
-AbstractCardDragItem::AbstractCardDragItem(AbstractCardItem *_item, const QPointF &_hotSpot, AbstractCardDragItem *parentDrag)
+AbstractCardDragItem::AbstractCardDragItem(AbstractCardItem *_item,
+                                           const QPointF &_hotSpot,
+                                           AbstractCardDragItem *parentDrag)
     : QGraphicsItem(), item(_item), hotSpot(_hotSpot)
 {
-    if (parentDrag) {
+    if (parentDrag)
```

Comment:
I thought we were putting open braces on the same line as control structures?

Reply:
if possible, lets leave it as is. I like them on separate lines

Reply:
There's a single option `AfterControlStatement: true` that defines the behaviour for all the "control statements", eg if/for/while/switch
I'm neutral on this, decide what you prefer.

Reply:
I'd strongly prefer using our existing style, keeping the brace on the same line as the control statement

Reply:
Since I’m one of the most active contributors currently, and all the work I’ve been doing follows what this PR change is, I’d really like to keep them on new lines if possible.

Reply:
Dont listen to @ZeldaZach , his style seems to change on a weekly basis :)

Reply:
Changed `AfterControlStatement: false` and pushed again